.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
Looking at https://github.com/electron/electron/labels, I see the following that should probably be exempt:

```
 - fixme/bug
 - fixme/crash
 - fixme/regression
 - fixme/security
 - blocked
 - blocking-stable
 - needs-review
 ```